recipes/dj-static/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "dj-static" %}
+{% set version = "0.0.6" %}
+{% set md5 = "078c4d5a455b7811173ffdb0cb33ed0f" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    md5: {{ md5 }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - static
+    run:
+        - python
+        - static
+
```

Comment:
@kwilcox add `pystache` as a dependency here and `conda-build-all` will know about it and will build it in the right order to avoid the failure we are seeing.